### ISSUE TITLE
Update Chinese (Simplified) translations

### ISSUE THREAD

**Issue by jasonren0403** [squidfunk/mkdocs-material#4956]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "zh",
  "action.edit": "编辑此页",
  "action.skip": "跳转至",
  "action.view": "查看本页的源代码",
  "announce.dismiss": "不再显示此消息",
  "blog.archive": "归档",
  "blog.categories": "分类",
  "blog.categories.in": "分类于",
  "blog.continue": "继续阅读",
  "blog.draft": "草稿",
  "blog.index": "回到主页",
  "blog.meta": "元数据",
  "blog.references": "相关链接",
  "clipboard.copy": "复制",
  "clipboard.copied": "已复制",
  "consent.accept": "同意",
  "consent.manage": "管理设定",
  "consent.reject": "拒绝",
  "footer": "页脚",
  "footer.next": "下一页",
  "footer.previous": "上一页",
  "header": "页眉",
  "meta.comments": "评论",
  "meta.source": "来源",
  "nav": "导航栏",
  "readtime.one": "需要 1 分钟阅读时间",
  "readtime.other": "需要 # 分钟阅读时间",
  "rss.created": "RSS 订阅",
  "rss.updated": "已更新内容的 RSS 订阅",
  "search": "查找",
  "search.placeholder": "搜索",
  "search.share": "分享",
  "search.reset": "清空当前内容",
  "search.result.initializer": "正在初始化搜索引擎",
  "search.result.placeholder": "键入以开始搜索",
  "search.result.none": "没有找到符合条件的结果",
  "search.result.one": "找到 1 个符合条件的结果",
  "search.result.other": "# 个符合条件的结果",
  "search.result.more.one": "在该页上还有 1 个符合条件的结果",
  "search.result.more.other": "在该页上还有 # 个符合条件的结果",
  "search.result.term.missing": "缺少",
  "select.language": "选择当前语言",
  "select.version": "选择当前版本",
  "source": "前往仓库",
  "source.file.contributors": "贡献者",
  "source.file.date.created": "创建日期",
  "source.file.date.updated": "最后更新",
  "tabs": "标签",
  "toc": "目录",
  "top": "回到页面顶部"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.

**Comment by squidfunk** [comment]
Thanks! Added in be0e198a4.

**Comment by squidfunk** [comment]
Released as part of [9.0.9](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.9)
